clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -409,7 +409,7 @@ class BinaryWrapper {
   std::pair<Constant *, Constant *>
   addStructArrayToModule(ArrayRef<Constant *> ArrayData, Type *ElemTy) {
 
-    auto *PtrTy = ElemTy->getPointerTo();
+    auto *PtrTy = llvm::PointerType::getUnqual(C);
```

Comment:
can any part of this go upstream? or is it all sycl specific?

Reply:
As far as I know, `ClangOffloadWrapper.cpp` is not present upstream, is it?

Reply:
oh sorry i thought it was `LinkerWrapper`, my bad